reference/features/README.md

```diff
@@ -9,14 +9,14 @@ Understanding our reference documentation is easier with background knowledge of
 [Authentication](/reference/features/authentication.md)
 [Configuration](/reference/features/configuration.md)
 [Distinct attribute](/reference/features/distinct.md)
-[Dumps](/reference/features/dumps.md)
+[Dumps](/learn/advanced/dumps.md)
```

Comment:
We should remove these links instead of updating them, since these pages are no longer in this site section.
(I know it's effectively pointless since these READMEs will be removed soon. Humor me 🥲 )